blivet/populator/populator.py

```diff
@@ -426,7 +426,7 @@ def _populate(self):
         self.drop_lvm_cache()
         mpath_members.drop_cache()
 
-        if flags.auto_dev_updates and not flags.image_install:
+        if flags.auto_dev_updates:
             blockdev.mpath.set_friendly_names(flags.multipath_friendly_names)
```

Comment:
This is the only place, where the flag `multipath_friendly_names` is used. So Anaconda might set `multipath_friendly_names` to False for the image installation.

Reply:
I would expect that multipath is not at all relevant for image installations since they use the disk images as an exclusive set of available/visible disks, meaning you can probably do whatever you want about multipath for image installations and it won't matter at all.